vpr/src/route/rr_graph_indexed_data.cpp

```diff
@@ -453,44 +409,41 @@ static void load_rr_indexed_data_T_values(int index_start,
                 // the combined capacitance of the node and internal capacitance of the switch. The
                 // second transient response is the result of the Rnode being distributed halfway along a
```

Comment:
Change "second transient response" to "multiplication by the second term by 0.5"